app/code/core/Mage/Api/Model/Server/Handler/Abstract.php

```diff
@@ -100,16 +115,6 @@ protected function _isAllowed($resource, $privilege = null)
         return $this->_getSession()->isAllowed($resource, $privilege);
     }
 
-    /**
-     *  Check session expiration
-     *
-     *  @return  bool
-     */
-    protected function _isSessionExpired()
-    {
-        return $this->_getSession()->isSessionExpired();
```

Comment:
Note: This line has a bug, `isSessionExpired()` expects a param `$user`:
https://github.com/OpenMage/magento-lts/blob/2a2a2fb504247e8966f8ffc2e17d614be5d43128/app/code/core/Mage/Api/Model/Session.php#L186-L186

I've checked, this function is not used.